ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -41,20 +45,25 @@ def add_arguments(self, parser, cli_name):
             parser, cli_name, '_verb', 'ros2doctor.verb', required=False)
 
     def main(self, *, parser, args):
+        print('Hello moto')
         """Run checks and print report to terminal based on user input args."""
         if hasattr(args, '_verb'):
             extension = getattr(args, '_verb')
             return extension.main(args=args)
 
+        # LocalVariables to reduce code length
```

Comment:
```suggestion
        # Local variables to reduce code length
```

Reply:
Absolutely, I am a bit embarassed by these mistakes, apologise me hehe. About the Flake8 error is because I left a blank space as other fragments do, such as https://github.com/ros2/ros2cli/blob/8e46bf2608d04e81a3d088ccc5087dbde9f3e32f/ros2doctor/ros2doctor/command/doctor.py#L54-L56
I can remove it but I think it is justified for clarity. I will make all these changes right now and push them. Thanks for the fb